plat/mediatek/mt6795/include/power_tracer.h

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Could headers like this be shared with mt8173 (and placed in plat/mediatek/common)?